store-conf/conf/log4j.properties

```diff
@@ -1,142 +1,240 @@
+logger.searchstat.level = info,SEARCHSTAT
+logger.searchstat.additivity = false
+logger.searchstat.appenderRef.SEARCHSTAT.ref = searchStatFile
+
+logger.ews.name = zimbra.ews
+logger.ews.level = info,EWS
+logger.ews.additivity = false
+logger.ews.appenderRef.EWS.ref = ewsFile
+
+logger.syslog.name = com.zimbra
+logger.syslog.level = debug
+logger.syslog.additivity = false
+logger.syslog.appenderRef.SYSLOG.ref = syslogAppender
 
 # HttpMethodBase spews out too many WARN on the badly formatted cookies.
-log4j.logger.org.apache.commons.httpclient.HttpMethodBase=ERROR
+logger.org.apache.commons.httpclient.HttpMethodBase=ERROR
 
 # spymemcached is too verbose at INFO level.
-log4j.logger.net.spy.memcached=WARN
-
-#log4j.logger.httpclient=DEBUG
-#log4j.logger.org.apache.commons.httpclient=DEBUG
-#log4j.logger.com.zimbra.common.util.ZimbraHttpConnectionManager=DEBUG
-
-#log4j.logger.zimbra.imap=DEBUG
-log4j.logger.zimbra.test=DEBUG
-
-log4j.logger.com.zimbra=INFO
-#log4j.logger.com.zimbra=DEBUG
-log4j.logger.com.zimbra.soap.SoapEngine=DEBUG
-#log4j.logger.com.zimbra.cs.store.BlobChunk=DEBUG
-#log4j.logger.com.zimbra.cs.Archiver=DEBUG
-#log4j.logger.com.zimbra.cs=DEBUG
-#log4j.logger.com.zimbra.cs.index=DEBUG
-
-#log4j.logger.zimbra.sqltrace=DEBUG
-log4j.logger.com.zimbra.cs.service.util.SpamHandler=DEBUG
-
-log4j.additivity.zimbra.sync=false
-log4j.logger.zimbra.sync=INFO,SYNC
-log4j.additivity.zimbra.synctrace=false
-log4j.logger.zimbra.synctrace=TRACE,SYNCTRACE
-log4j.additivity.zimbra.syncstate=false
-log4j.logger.zimbra.syncstate=INFO,SYNCSTATE
-log4j.additivity.zimbra.wbxml=false
-log4j.logger.zimbra.wbxml=INFO,WBXML
-log4j.additivity.zimbra.activity=false
-log4j.logger.zimbra.activity=INFO,ACTIVITY
-log4j.additivity.zimbra.ews=false
-log4j.logger.zimbra.ews=INFO,EWS
-log4j.logger.zimbra.doc=DEBUG
-log4j.logger.zimbra.dav=DEBUG
-log4j.logger.zimbra.offline=DEBUG
-log4j.logger.zimbra.offline.request=INFO
-log4j.logger.zimbra.offline.response=INFO
-log4j.logger.zimbra.webclient=DEBUG
-log4j.logger.zimbra.slogger=OFF
-log4j.logger.zimbra.searchstat=INFO,SEARCHSTAT
+logger.net.spy.memcached=WARN
+
+logger.slogger.name = zimbra.slogger
+logger.slogger.level=INFO,SLOGGER
+#logger.slogger.level=ERROR
+logger.slogger.additivity = false
+logger.slogger.appenderRef.SLOGGER.ref = sloggerAppender
+
+logger.soapengine.name = com.zimbra.soap.SoapEngine
+logger.soapengine.level = debug
+
+logger.blobchunk.name = com.zimbra.cs.store.BlobChunk
+logger.blobchunk.level = debug
+
+logger.zimbracs.name = com.zimbra.cs
+logger.zimbracs.level = debug
+
+logger.zimbradoc.name = zimbra.doc
+logger.zimbradoc.level = debug
+
+logger.zimbraoffline.name = zimbra.offline
+logger.zimbraoffline.level = debug
+
+logger.offlinerequest.name = zimbra.offline.request
+logger.offlinerequest.level = debug
+
+logger.offlineresponse.name = zimbra.offline.response
+logger.offlineresponse.level = debug
+
+logger.webclinet.name = zimbra.webclient
+logger.webclinet.level = debug
```

Comment:
are these going to be default levels?

Reply:
No.. This file can be used just as a template.

Reply:
No.. This file can be used just as a template.